File: codechef/JUNE20/B.cpp
//“Make it work, make it right, make it fast.” – Kent Beck
#include<bits/stdc++.h>
using namespace std;
typedef long long ll;
typedef double dd;
const int siz = 1e6 + 5;
const int MOD = 1e9 + 7;
#define endl '\n'
#define deb(x) cout << #x << " = " << x << endl;

void solve(){
  string s;
  cin >> s;
  int count = 0;
  int n = (int)s.size();
  for(int i = 0; i < n-1; i++){
    if((s[i] == 'x' and s[i+1] == 'y') or (s[i] == 'y' and s[i+1] == 'x'))
        count++, i++;
  }
  cout << count << endl;
}

int main(){
  #ifndef RAY
    //freopen("input.txt", "r", stdin);
  #endif
    ios::sync_with_stdio(false);
    cin.tie(nullptr);
    int t = 1;
    cin >> t;
    while(t--){
      solve();
    }
  return 0;
}  

File: codechef/JUNE20/B.py
t = input(int)
# print(type(t))
while t > 0 :
	t -= 1
	s = raw_input()
	n = len(s)
	# print(n)
	ans = 0
	i = 0
	# print(type(s))
	while i  < (n-1):
		# print(s[i], s[i+1])
		if (s[i+1] == 'x' and s[i] == 'y' ) or (s[i] == 'x' and s[i+1] == 'y' ) :
			ans += 1
			i += 1
		i += 1
	print(ans)

